app/src/main/java/com/github/epfl/meili/home/GoogleSignInActivity.kt

```diff
@@ -28,6 +29,11 @@ class GoogleSignInActivity : AppCompatActivity() {
         updateUI()
     }
 
+    fun OnMapViewButtonClick(@Suppress("UNUSED_PARAMETER") view: View) {
```

Comment:
Nitpick: function names should start with a lowercase.

Reply:
Changed 👍